### PR TITLE
feat: Kibbeh added basic ability to check upload in ChangeAvatarcard

### PR DESCRIPTION
**Pull request by AkshayJaitly**
In this pr:

- Added code to upload files and test functionality of change avatar card
- code cleanup and formatting
- removed unused imports
- added comments for future scope
- added images/jpg type and handled some edge cases with file size

### CODE REVIEW

File: kibbeh/src/ui/ChangeAvatarCard.tsx [hunk 1]
```diff
@@ -1,59 +1,82 @@
...
-   );
-}
+  const [baseAvatarUrl, setAvatarUrl] = useState(avatarUrl);
+
+  const handleImageUpload = (e: React.ChangeEvent<HTMLInputElement>) => {
+    const MAX_FILE_SIZE = 3145728;
+    const file = (e.target.files as FileList)[0];
+
+    if (file.size > MAX_FILE_SIZE) {
+      console.log("File size must be less than 3MB");
```
Comment (wes337) [inline kibbeh/src/ui/ChangeAvatarCard.tsx]: Nit: should we avoid console log in code for production?

Comment (AkshayJaitly) [reply kibbeh/src/ui/ChangeAvatarCard.tsx]: sure let me get rid of it, makes sense

**Comment by goldyydev** [comment]
Do you mind updating the linting tests?

**Comment by AkshayJaitly** [comment]
sure, what should I do to fix linting tests?

**Comment by AkshayJaitly** [comment]
apparently there is a linting issue in useDefStore and SingleUser tax, both which I am not updating but are like that from before.

**Comment by amitojsingh366** [comment]
use proper commit messages please *(`yarn commit` or `yarn commit:signed`)*
and the setDeaf etc will be fixed if you pull latest and push

**Comment by AkshayJaitly** [comment]
I have used yarn commit

**Comment by amitojsingh366** [comment]
> 
> 
> I have used yarn commit

all you commits need to be yarn commit and the scope can only be from one of these below
image

**Comment by AkshayJaitly** [comment]
ok let me open a new pr
